kotlin-result/build.gradle.kts

```diff
@@ -52,9 +46,6 @@ kotlin {
         }
 
         val jvmMain by getting {
-            dependencies {
-                implementation(kotlin("stdlib-jdk8"))
-            }
         }
```

Comment:
I think we can remove this whole lambda now, given its empty

Reply:
done